core/core.go

```diff
@@ -25,6 +25,7 @@ var (
 	errAlreadyConfigured = errors.New("core is already configured; must reset first")
 	errUnconfigured      = errors.New("core is not configured")
 	errNoMockHSM         = errors.New("core is not configured with a mockhsm")
+	errMockHSM           = errors.New("starting core with mockhsm: should only be used in development")
```

Comment:
A few minor things: 

1. It's overkill to make this an `error`. Since we only ever log it's string representation, we don't need to make it an error. Let's just log the string inline. 
2. Since this message shows not when the core starts, but when the cluster is initialized, what about a message like: "warning: this core uses a mockhsm. mockhsm data does not sync across coreds" (Especially since that really is the issue we're concerned about.)